packit_service/worker/handlers/github_handlers.py

```diff
@@ -380,7 +383,10 @@ def __init__(self, config: ServiceConfig, event: PullRequestCommentEvent):
     def run(self) -> HandlerResults:
         collaborators = self.project.who_can_merge_pr()
         if self.event.github_login not in collaborators | self.config.admins:
-            msg = "Only collaborators can trigger Packit-as-a-Service"
+            msg = (
+                "Only users with write or admin permissions to the repository "
```

Comment:
I am only a little confused about the `write permissions` meaning. (Since in the comments, you can see terms like `member`, `collaborator`, ...)

Reply:
If I understand it correctly than `member` and `collaborator` are org membership terms (roles?), but we check for [repository permission levels](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization).

(Yes, this part of GitHub is very confusing.)